### DISCUSSION TITLE
Portal 2 and demon slayer

### DISCUSSION THREAD

**Discussion by PlayGame666** [skyline-emu/skyline#1969] [Q&A] [answered]
![Screenshot_2022-11-09-09-53-44-360_skyline emu](https://user-images.githubusercontent.com/107650760/200770829-27a8242a-ad91-4b14-9a92-e4254356d857.jpg)
when will you be able to run Portal 2 on your emulator? Because I can't even hit the play button. This is the same problem with the game Demon slayer kimetsu no yaiba the hinokami chronicles. wrote through a translator

**Comment by Crytonics** [comment, marked as answer]
You have old keys or broken, you need to re-dump them with newer version. Rn Portal 2 doesen't work yet.
